assets.css

```
@@ -2,7 +2,33 @@
  * Restrict sizing to the page width in all browsers (opinionated).
  */
 
-:where(iframe, img, input, video, select, textarea) {
+:where(iframe, input, video, select, textarea) {
   height: auto;
   max-width: 100%;
 }
+
+/**
+ * Do not stretch image or grow bigger than parent
+ */
+
+:where(img) {
+  max-width: 100%;
+}
+
+/**
+ * Preserve aspect ratio
+ */
+
+:where(img[width][height]) {
+  height: auto;
+}
```

Would it work to simplify this strategy by changing the initial selector?

```suggestion
:where(iframe, img[width][height], input, video, select, textarea) {
```

---

Then we miss the img where no width and height is set. We could do this:

```
:where(iframe, img[width][height], input, video, select, textarea) {
  height: auto;
}

:where(iframe, img, input, video, select, textarea) {
  max-width: 100%;
}
```